src_py/nerlPlanner/JsonElementWorkerDefinitions.py

```diff
@@ -8,7 +8,7 @@
 LayerTypeMap = OrderedDict([
     ("Default" , "0"),
     ("Scaling" , "1"),
-    ("Conv" , "2"),
+    ("CNN" , "2"),
```

Comment:
Why do you change conv layer to CNN?
Conv is the correct name of convolutional layer.